kolibri/core/discovery/api.py

```diff
@@ -47,6 +71,7 @@ def get_object(self, id_filter=None):
         for filter_key in ("id", "instance_id"):
             try:
                 obj = queryset.get(**{filter_key: id_filter})
+                obj.dynamic = obj.location_type == LocationTypes.Dynamic
```

Comment:
Unless I'm misunderstanding, it seems we should adjust usages of `dynamic` if the reason for this is to ensure it's returned from the API. If it's necessary, you could add the type field so decisions can be based off that instead.

Reply:
Could you explain what you mean by “adjust usages of `dynamic`”?

Reply:
Sorry, I presumed this was to maintain some usages of `dynamic` coming through the API, so I meant "refactor those to _not_ use `dynamic`". Is that the case?

Reply:
I believe that was the case. Would it be preferred to just make sure the `location_type` field is included in the API response instead?

Reply:
@bjester?

Reply:
Sorry for the delay here. Yes, if it's reasonable to update frontend usages of `dynamic` to use `location_type` instead, that would be great. It should also uncover any potential issues where the new reserved type could get mixed up